.env

```
@@ -0,0 +1,5 @@
+# Identifiants PostgreSQL (Local Dev)
+POSTGRES_USER=dev_user
+POSTGRES_PASSWORD=dev_password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)